readme.md

```diff
@@ -68,18 +68,25 @@ Show DevTools on each created `BrowserWindow`.
 
 ### devTools([window])
 
-Type: `BrowserWindow`<br>
-Default: the focused `BrowserWindow`
-
 Toggle DevTools for the specified `BrowserWindow` instance or the focused one.
 
-### refresh([window])
+#### options
+
+##### window
 
 Type: `BrowserWindow`<br>
 Default: the focused `BrowserWindow`
 
+### refresh([window])
+
 Reload the specified `BrowserWindow` instance or the focused one.
 
+#### options
```

Comment:
This is incorrect. It's not an option, but an argument. The existing definition was correct.